src/screens/TopicMemberScreen/index.js

```diff
@@ -89,15 +89,15 @@ const TopicMemberScreen = () => {
     const result = await getAllMemberTopic(query);
     if (result.code === 200) {
       const newDataFollowed = result.data
-        .filter((item) => item.is_following && profile.myProfile.user_id !== item.user_id)
+        .filter((item) => item.is_following)
         .map((data) => ({
           ...data,
           name: data.username,
           image: data.profile_pic_path,
           description: null
         }));
       const newDataUnfollowed = result.data
-        .filter((item) => !item.is_following && profile.myProfile.user_id !== item.user_id)
+        .filter((item) => !item.is_following)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the removal of the condition `profile.myProfile.user_id !== item.user_id` simplifies the filter operation. Please ensure that this change does not introduce any unintended side effects.

```diff
- .filter((item) => !item.is_following && profile.myProfile.user_id !== item.user_id)
+ .filter((item) => !item.is_following)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->